### ISSUE TITLE
Use lang attribute to mark changes in language.

### ISSUE THREAD

**Issue by joedolson** [WordPress/Learn#976]
## Description

On the Browse Lesson Plans & single-lesson plan pages; there's no language of parts defined for non-English lessons. E.g. [https://learn.wordpress.org/lesson-plan/%ce%b5%ce%b9%cf%83%ce%b1%ce%b3%cf%89%ce%b3%ce%[…]%84%cf%8d%cf%80%ce%bf%cf%85%cf%82-%cf%80%cf%81%ce%bf%cf%83/](https://learn.wordpress.org/lesson-plan/%ce%b5%ce%b9%cf%83%ce%b1%ce%b3%cf%89%ce%b3%ce%ae-%cf%83%cf%84o%cf%85%cf%82-%cf%80%ce%b9%ce%bf-%ce%ba%ce%bf%ce%b9%ce%bd%ce%bf%cf%8d%cf%82-%cf%84%cf%8d%cf%80%ce%bf%cf%85%cf%82-%cf%80%cf%81%ce%bf%cf%83/) or the first course listed at https://learn.wordpress.org/lesson-plans/?_view=all are lessons in Greek; but they're on an English language page, so the screen reader doesn't know how to pronounce them. For Greek, this results in spelling each word out (Epsilon iota sigma, etc.); in a language using the roman alphabet it would just be really bad pronunciation. It would be good if non-English language strings were wrapped in appropriate language attributes.

## Step-by-step reproduction instructions

Visit any page with mixed languages.
Activate a screen reader. (E.g., NVDA, VoiceOver)
Attempt to read the content.

## Expected behaviour

Strings should be pronounced according to the language they are written in.

## Actual behaviour

Strings are pronounced using the phonemes of the page's global lang attribute.

## Environment information

This is expected behavior when the language is not marked correctly using a lang attribute. However, for reproduction it's important to note that not all voice synthesizers handle language changes on the fly, so even when correctly marked up, the results may vary depending on which screen reader is used *and* which voice synthesizer is configured in that screen reader.

**Comment by adamwoodnz** [comment]
Thanks for raising this @joedolson 

> It would be good if non-English language strings were wrapped in appropriate language attributes.

Could you please describe the fix in more detail please? What would this look like in terms of code changes?

**Comment by joedolson** [comment]
When text is not in English, it needs to be wrapped in a container that indicates what language it is in, e.g.

```<span lang="es">Hola!</span> <span lang="de">Sprechen sie Deutsch?</span>```

The language of the whole page is set in the `html` element, and that's the language a screen reader will use when reading the page. When the non-English parts of the page are encountered, however, the screen reader will not know how to handle them unless the new language is indicated.

**Comment by adamwoodnz** [comment]
Related: https://wptavern.com/two-new-wordpress-plugins-improve-block-editor-accessibility-and-wcag-compliance
